bindings/pydeck-carto/pydeck_carto/credentials.py

```diff
@@ -0,0 +1,38 @@
+import json
+import requests
+
+
+def load_carto_credentials(filepath):
+    """Load credentials from file. It must contain the following attributes:
+    - api_base_url
```

Comment:
Not for right now, but a great feature would be that a user could get the accessToken through the browser without requiring to create an app at CARTO dashboard.

As google collab does:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/1161870/173618048-f9358ac5-f072-4225-9cf6-72083dab12c3.png">